test/e2e_node/e2e_node_suite_test.go

```diff
@@ -59,6 +59,9 @@ import (
 	_ "k8s.io/kubernetes/test/e2e/framework/node/init"
 	_ "k8s.io/kubernetes/test/utils/format"
 
+	gcemetadata "cloud.google.com/go/compute/metadata"
+	_ "k8s.io/kubernetes/test/e2e/framework/providers/gce"
```

Comment:
I thought we were trying to remove provider-specific code and tests... this looks like it's going in the opposite direction and linking this into a new binary / test